tests/plugins/clipboard/_helpers/pasting.js

```diff
@@ -169,7 +169,7 @@ function testResetScenario( editor, queue ) {
 function getDefaultNotification( editor, command, keyInfo ) {
 	var keystroke = keyInfo || CKEDITOR.tools.keystrokeToString( editor.lang.common.keyboard,
 		editor.getCommandKeystroke( editor.commands[ command ] ) ),
-		msg = editor.lang[ command === 'paste' ? 'clipboard' : command ].pasteMsg
+		msg = ( command === 'paste' ? editor.lang.clipboard.pasteDefaultMsg : editor.lang[ command ].pasteMsg )
```

Comment:
`editor.lang[ command ].pasteMsg` should be also reflected with property change.